.travis.yml

```diff
@@ -30,15 +30,8 @@ matrix:
       env: DB=MYSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1 PDO=1
     - php: 7.4
       env: DB=MYSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
-
-    # Warning: phpunit with --coverage-clover takes too much time, so travis build fails after 10
-    # minutes with no output from the task.
-    # We have to disable it until we come up with another (faster) solution to gather coverage
-    #
-    # See https://github.com/silverstripe/silverstripe-assets/pull/407
-    #
-    # - php: 7.3
-    #   env: DB=MYSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_COVERAGE_TEST=1
+    - php: 7.3
```

Comment:
we already have `php: 7.3` above in the matrix. Might be worth adding coverage there instead

Reply:
Historically, we tend to have code coverage on its own matrix entry ... although there's no specific reason for doing this.

Reply:
true. ok then, let's do it